server/app/routes/api/wines.js

```diff
@@ -45,33 +44,29 @@ router.get('/:id/rating', function(req, res) {
 
 //on accesssible to admins
 //perhaps need to consider preventing duplicate posts
```

Comment:
I think we can try to enhance this later. Just left a comment for us to revisit.